src/events.js

```diff
@@ -13,16 +13,17 @@ export const paneConnected = "l:pane:connected";
 /**
  * @param {HTMLElement} el
  * @param {Evented} evented
+ * @param bubble determines whether events should bubble (defaults to true)
  */
-export const connectLeafletEvents = (el, evented) => {
+export const connectLeafletEvents = (el, evented, bubble = true) => {
```

Comment:
Maybe call this parameter `bubbles` to match the value in the `CustomEvent` signature?

Reply:
As discussed, I only called it bubble because it applies to all events created under this function.